README.md

```diff
@@ -37,11 +37,15 @@ In order to verify the email for a new subscription:
 
 This allows the user to list, modify and delete their subscriptions [[login](https://www.gov.uk/email/manage/authenticate)]. It uses a similar, but separate email/token process to authenticate a user, establishing a session for them to make their changes.
 
+### Testing 
+
+[Receive emails from Email Alert API in Integration and Staging](https://docs.publishing.service.gov.uk/manual/receiving-emails-from-email-alert-api-in-integration-and-staging.html)
```

Comment:
+1 on including the docs here. We could help future people by explaining why this is useful:

```suggestion
Some pages are only accessible from a link sent in an email. The documentation on [receiving emails from Email Alert API in Integration and Staging](https://docs.publishing.service.gov.uk/manual/receiving-emails-from-email-alert-api-in-integration-and-staging.html) will show how to send emails so these pages can be viewed.```